internal/frontend/handler.go

```diff
@@ -110,7 +110,7 @@ func (handler *StaticHandler) serveFile(w http.ResponseWriter, r *http.Request,
 		} else {
 			w.Header().Set("Referrer-Policy", "strict-origin-when-cross-origin")
 
-			w.Header().Set("Content-Security-Policy", "default-src 'self' https://*.userpilot.io; img-src * data: ; "+
+			w.Header().Set("Content-Security-Policy", "script-src 'self' https://*.userpilot.io 'unsafe-inline'; img-src * data: ; "+
```

Comment:
the script-src/default-src change shouldnt change anything (default is the faullback if no script, and we have no script)

what problem are you seeing @rothgar? cause `unsafe-inline` is bit _meh_ - unlikely to be an issue for us specifically but if avoidable preferable to avoid